__tests__/app.test.js

```diff
@@ -40,3 +41,45 @@ describe("3. GET /api/topics", () => {
       });
   });
 });
+describe("4.GET /api/articles", () => {
+  test("responds with statusCode 200 and also if the response contains an object with an array of articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
```

Comment:
we don't need this line if we're successfully looping over the array in the next line, we do however need to check that it's not empty to avoid getting a false positive test